virttest/utils_config.py

```diff
@@ -1,8 +1,10 @@
 import ast
 import logging
 import os.path
-import platform
 from six import StringIO
+
+from avocado.utils import distro
```

Comment:
let us import avocado module after built-in.

Reply:
sure